### DISCUSSION TITLE
retryFailedStep plugin does not work on CodeceptJS version 2.6.2

### DISCUSSION THREAD

**Discussion by johnny-gottschalk** [codeceptjs/CodeceptJS#3026] [Q&A]
I would like to use the retryFailed plugin for my automated tests in CodeceptJS 2.6.2.

I have set up the retryFailed plugin in the following way:

`  retryFailedStep: {
            enabled: true,
            minTimeout: 1500,
        },`

When I excute CodeceptJS with:

`npx codeceptjs run --plugins retryFailedStep --config=./config/local.config.js` 

On a Windows Machine. Failing steps are never retried and executed again.

### Details

* CodeceptJS version: 2.6.2
* NodeJS Version: v12.16.3
* Operating System: Windows 10
*  webdriverio

**Comment by mangel0111** [comment]
It doesn't work either on the 3.0.0 version.

**Comment by AndiLavera** [comment]
And here I thought I was going crazy on 3.3.0

**Reply by AramSol** [reply]
this not working for me too, codeceptjs 3.3.0
